app/lib/test_applications.rb

```diff
@@ -204,6 +208,26 @@ def confirm_enrollment(choice)
     end
   end
 
+  def add_note(choice)
+    provider_user = choice.provider.provider_users.first
+    provider_user ||= add_provider_user_to_provider(choice.provider)
+    as_provider_user(choice) do
+      @time = @time + rand(-5..5).days
```

Comment:
Could we reuse (or make some extensions to) `fast_forward` here?

Reply:
`fast_forward` mainly deals with the timestamps of audited events. I've changed the code to use `travel_to`, which is used elsewhere, too.